packages/web-components/src/textarea/textarea.stories.ts

```diff
@@ -0,0 +1,285 @@
+      control: 'text',
+    },
+    defaultContent: {
+      description: 'Content inside the component',
+      control: 'text',
+    },
+    appearance: {
+      description: 'Sets the visual appearance of the control',
+      table: {
+        defaultValue: { summary: `${TextAreaAppearance.outline}` },
+      },
+      control: 'select',
+      options: Object.values(TextAreaAppearance),
+    },
+    autocomplete: {
+      description: "The element's autocomplete state.",
+      control: 'text',
+    },
+    autofocus: {
+      description: 'Whether this element should get focus after the page finishes loading.',
+      control: 'boolean',
+    },
+    autoResize: {
+      description: 'Whether the element’s height should be automatically changed based on the content.',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+      control: 'boolean',
+    },
+    block: {
+      description: 'Whether the element should be a block-level element.',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+      control: 'boolean',
+    },
+    dirname: {
+      description: 'Sets the directionality of the element to be submitted with form data.',
+      control: 'text',
+    },
+    disabled: {
+      description: 'Sets the disabled state',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+      control: 'boolean',
+    },
+    displayShadow: {
+      description: 'Whether the element displays a visual box shadow',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+      control: 'boolean',
+    },
+    form: {
+      description: 'The id of a form to associate the element to.',
+      control: 'text',
+    },
+    maxLength: {
+      description: 'Sets the maximum number of characters allowed in the input',
+      control: 'number',
+    },
+    minLength: {
+      description: 'Sets the minimum number of characters allowed in the input',
+      control: 'number',
+    },
+    name: {
+      description: 'Sets the name of the input',
+      control: 'text',
+    },
+    placeholder: {
+      description: 'Sets the placeholder text',
+      control: 'text',
+    },
+    readOnly: {
+      description: 'Sets the readonly state',
+      control: 'boolean',
+    },
+    required: {
+      description: 'Sets the required state',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+      control: 'boolean',
+    },
+    resize: {
+      description: 'Whether and how a user can resize the element.',
+      control: 'select',
+      options: Object.values(TextAreaResize),
+    },
+    size: {
+      description: 'Sets the size of the control',
+      control: 'select',
+      options: Object.values(TextAreaSize),
+    },
+    spellcheck: {
+      description: 'Controls whether to enable spell checking for the content.',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+      control: 'boolean',
+    },
```

Comment:
I realize it's because we're mirroring the `<textarea>` API, but that's a lotta options! 😅

Reply:
I know right 😄?! Also, the design spec has a bunch as well. So, they all add up.